test/e2e-go/cli/goal/expect/goalExpectCommon.exp

```diff
@@ -453,7 +453,7 @@ proc ::AlgorandGoal::WaitForAccountBalance { WALLET_NAME ACCOUNT_ADDRESS EXPECTE
                 puts "Account balance OK: $ACCOUNT_BALANCE"; break
             } else {
                puts "Account balance: '$ACCOUNT_BALANCE' does not match expected balance: '$EXPECTED_BALANCE'"
-               if { $i >= 10 } then { ::AlgorandGoal::Abort "Account balance $ACCOUNT_BALANCE does not match expected amount: $EXPECTED_BALANCE"; break;}
+               if { $i >= 20 } then { ::AlgorandGoal::Abort "Account balance $ACCOUNT_BALANCE does not match expected amount: $EXPECTED_BALANCE"; break;}
```

Comment:
Maybe make these two messages different? The non-error case might end with ", still waiting..." ? The timeout case might end with, ", waited too long, FAIL" ?

Reply:
Good thinking, I updated the messaging.